core/src/com/unciv/models/ruleset/Unique.kt

```diff
@@ -7,23 +7,45 @@ import com.unciv.logic.map.MapUnit
 import com.unciv.logic.map.TileInfo
 import com.unciv.models.stats.Stat
 import com.unciv.models.stats.Stats
-import com.unciv.models.translations.fillPlaceholders
-import com.unciv.models.translations.getPlaceholderParameters
-import com.unciv.models.translations.getPlaceholderText
-import com.unciv.models.translations.hasPlaceholderParameters
+import com.unciv.models.translations.*
 import com.unciv.ui.worldscreen.unit.UnitActions
 import kotlin.random.Random
 
-class Unique(val text:String){
-    val placeholderText = text.getPlaceholderText()
-    val params = text.getPlaceholderParameters()
+class Unique(val text: String){
     /** This is so the heavy regex-based parsing is only activated once per unique, instead of every time it's called
      *  - for instance, in the city screen, we call every tile unique for every tile, which can lead to ANRs */
+    val placeholderText = text.getPlaceholderText()
+    val params = text.getPlaceholderParameters()
     val stats: Stats by lazy {
         val firstStatParam = params.firstOrNull { Stats.isStats(it) }
         if (firstStatParam == null) Stats() // So badly-defined stats don't crash the entire game
         else Stats.parse(firstStatParam)
     }
+    val conditionals: List<Unique> = text.getConditionals()
+    
+    
+    // This function will get LARGE, as it will basically check for all conditionals if they apply
+    // This will require a lot of parameters to be passed (attacking unit, tile, defending unit, civInfo, cityInfo, ...)
+    // I'm open for better ideas, but this was the first thing that I could think of that would
+    // work in all cases.
+    fun conditionalsApply(civInfo: CivilizationInfo? = null): Boolean {
+        for (condition in conditionals) {
+            if (!conditionalApplies(condition, civInfo)) return false
+        }
+        return true
+    }
+    
+    private fun conditionalApplies(
+        condition: Unique, 
+        civInfo: CivilizationInfo? = null
+    ): Boolean {
+        when (condition.placeholderText) {
+            "when not at war" -> if (civInfo == null || civInfo.isAtWar()) return false
+            "when at war" -> if (civInfo == null || !civInfo.isAtWar()) return false
+            else -> return false
+        }
+        return true
+    }
```

Comment:
Can't help it, my fingers are typing without my conscious support:
```kotlin
) = when (condition.placeholderText) {
            "when not at war" -> civInfo?.civInfo.isAtWar() == false
            "when at war" -> civInfo?.civInfo.isAtWar() == true
            else -> false
        }
```
..makes no bytecode difference.

Reply:
The lack of conscious support was clear from the doubling of the `civInfo` :p
But in the end that would probably look cleaner, so I'll use this as inspiration

Reply:
> inspiration

wasn't meant as anything other

(yay! double negative!)